### DISCUSSION TITLE
newsletter feature

### DISCUSSION THREAD

**Discussion by PriyansuMaurya** [UniKonf/vibey#45] [Q&A]
Hey. I was looking at the implementation of the newsletter feature, is it implemented or not?

**Comment by mkubdev** [comment]
@PriyansuMaurya Hi! 👋🏻 
No, the feature is not yet implemented, do you to work on this feature? Feel free to open a issue/PR!

**Reply by PriyansuMaurya** [reply]
Would love to.

But I want to implement the same thing on my website.

If by fortune i am able to succeed 🙂 then definitely.

**Reply by mkubdev** [reply]
Super cool! What do you plan to use as third party for the newsletter ?

**Comment by PriyansuMaurya** [comment]
I am still trying to determine the best path forward and if you have any suggestions or recommendations, I would greatly appreciate it.

**Reply by mkubdev** [reply]
I'm not so aware of all newsletetr tech available. I know [**Listmonk**](https://listmonk.app/), a self-hosted newsletter service but that's not what we aim.

Maybe using Sendinblue, Mailchimp.. 

I found this article too: https://mikebifulco.com/posts/publish-your-newsletter-with-convertkit-api-next-js

**Reply by mkubdev** [reply]
It's better to use a thirdparty newsletter provider ig
